converters/toZigbee.js

```diff
@@ -26,8 +26,20 @@ const converters = {
     read: {
         key: ['read'],
         convertSet: async (entity, key, value, meta) => {
-            const result = await entity.read(value.cluster, value.attributes);
+            const result = await entity.read(value.cluster, value.attributes, (value.hasOwnProperty('options') ? value.options : {}));
             meta.logger.info(`Read result of '${value.cluster}': ${JSON.stringify(result)}`);
+            if (value.hasOwnProperty('store')) {
+                const state = {};
+                state[value.store] = result;
+                return {state: state};
+            }
+        },
+    },
+    write: {
+        key: ['write'],
+        convertSet: async (entity, key, value, meta) => {
+            await entity.write(value.cluster, value.payload, utils.getOptions(meta.mapped, entity));
```

Comment:
Please also allow to pass options here similar to the `read`.

Reply:
Do we merge them with the getOptions or do we pass them in favor of getOptions?

Reply:
Something like the new commit?

Reply:
Looks good, thanks!